src/_assets/css/main.scss

```diff
@@ -815,51 +802,6 @@ main .content {
   border-radius: 8px;
 }
 
-.codesample__open-in-dartpad {
-  display: block;
-  position: absolute;
-  width: auto;
-  z-index: 10;
-  padding: 5px 10px 5px 35px;
-  top: 35px;
-  right: 38px;
-  background-color: $gray;
-  color: $gray-base;
-
-  font-weight: normal;
-  font-size: $font-size-base * 0.91;
-  @include media-breakpoint-up(md) {
-    font-size: $font-size-base * 0.8;
-  }
-  @include media-breakpoint-up(lg) {
-    font-size: $font-size-base * 0.91;
-  }
-
-  &.top {
-    top: 0;
-  }
-  &:visited, &:active {
-    color: $gray-base;
-  }
-  &:hover {
-    background-color: darken($gray, 10%);
-    color: $white-base;
-  }
-  &:after {
-    content: '';
-    display: block;
-    position: absolute;
-    top: 5px;
-    left: 10px;
-    width: 18px;
-    height: 18px;
-    background-image: asset_url('shared/dart/logo/default.svg');
-    background-size: cover;
-    background-position: center center;
-    background-repeat: no-repeat;
-  }
-}
-
```

Comment:
This CSS class was introduced July 11, 2016, [Migrating repo from Ocupop](https://github.com/dart-lang/site-www/commit/fde624e85deceb0590beeb5ab65a06f8e31a1ffa). AFAICT, since the Dash migration, this style isn't used anymore (I've checked the site sources and the generated site).